omero/developers/Clients/ImportLibrary.txt

```diff
@@ -55,7 +55,7 @@ data files are no longer duplicated and any nested directory structure is
 preserved. It also allows OMERO to take advantage of pre-generated pyramids
 available in some formats e.g. SVS.
 
-For full details of the import workflow see :doc:`/developers/ManagedRepository/ManagedRepository`.
+For full details of the import workflow see :doc:`/developers/ImportFS`.
```

Comment:
The benefit of the naming "ManagedRepository" is because it makes it clear that the mrepo is a significant component like dropbox which needs configuration, etc. (Not that there's not *also* a need for an "Import" page)